dotnetv3/S3/README.md

```diff
@@ -16,51 +16,45 @@ Amazon S3 is storage for the internet. You can use Amazon S3 to store and retrie
+- [Get CORS rules for a bucket](s3CORSExample/S3CORS.cs) (`GetCORSConfigurationAsync`)
+- [Get an object from a bucket](S3_Basics/S3_Basics.cs) (`GetObjectAsync`)
 - [Get the ACL of a bucket](BucketACLExample/BucketACL.cs) (`GetBucketAclAsync` )
+- [Get the lifecycle configuration of a bucket](LifecycleExample/Lifecycle.cs) (`GetLifecycleConfigurationAsync`)
+- [Get the website configuration for a bucket](WebsiteConfigExample/WebsiteConfig.cs) (`GetBucketWebsiteAsync`)
 - [List buckets](ListBucketsExample/ListBuckets.cs) (`ListBucketsAsync`)
-- [List objects in a bucket](ListObjectsExample/ListObjects.cs) (`ListObjectsV2Async`)
-- [List objects in a bucket using a dual stack endpoint](DualStackEndpointExample/DualStackEndpoint.cs) (`ListObjectsV2Async`)
+- [List object versions in a bucket](versioned-examples/ListObjectVersionsExample/ListObjectVersions.cs) (`ListObjectVersionsAsync`)
+- [List objects in a bucket](S3_Basics/S3_Basics.cs) (`ListObjectsV2Async`)
 - [List objects in a bucket using a paginator](ListObjectsPaginatorExample/ListObjectsPaginator.cs) (`ListObjectsV2Async`)
-- [List the versions of an object](versioned-examples/ListObjectVersionsExample/ListObjectVersions.cs) (`ListObjectVersionsAsync`)
-- [Log server access](ServerAccessLoggingExample/ServerAccessLogging.cs) (`PutACLAsync`, `PutBucketLoggingAsync`)
-- [Manage a bucket's ACL](ManageACLsExample/ManageACLs.cs) (`PutBucketAsync`, `GetACLAsync`, `PutACLAsync`)
-- [Manage an object's ACL](ManageObjectACLExample/ManageObjectACL.cs) (`PubtObjectACLAsync`, `GetObjectAclAsync`)
-- [Manage bucket CORS rules](s3CORSExample/S3CORS.cs) (`GetBucketCorsAsync`, `PutBucketCorsAsync`, `DeleteBucketCorsAsync`)
-- [Manage the lifecycle of a bucket](LifecycleExample/Lifecycle.cs) (`GetBucketLifecycleConfigurationAsync`, `PutBucketLifecycleConfigurationAsync`, `DeleteLifecycleConfigurationAsync`)
-- [Perform a multi-part copy](SSECLowLevelMPUcopyObjectExample/SSECLowLevelMPUcopyObject.cs) (`InitiateMultipartUploadAsync`, `UploadPartAsync`, `CompleteMultipartUploadAsync`)
-- [Perform a multi-part upload](MPUapiCopyObjExample/MPUapiCopyObj.cs) (`InitiateMultipartUploadAsync`, `CopyPartAsync`, `CompleteMultipartUploadAsync`)
-- [Perform a multi-part upload of a directory](UploadDirMPUHighLevelAPIExample/UploadDirMPUHighLevelAPI.cs) (`UploadDirectoryAsync`)
-- [Perform a multi-part upload of a file](UploadFileMPUHighLevelAPIExample/UploadFileMPUHighLevelAPI.cs) (`UploadAsync`)
-- [Perform a multi-part upload of a file](UploadFileMPULowLevelAPIExample/UploadFileMPULowLevelAPI.cs) (`UploadPartAsync`)
-- [Restore an archived object](RestoreArchivedObjectExample/RestoreArchivedObject.cs) (`RestoreObjectAsync`)
+- [List objects using temporary credentials](TempCredExplicitSessionStartExample/TempCredExplicitSessionStart.cs)
+- [List objects using temporary federated credentials](TempFederatedCredentialsExample/TempFederatedCredentials.cs)
```

Comment:
Normally this list should exactly match the list in the TOC, but I see that these are sub-examples in a relatively complex example, so I think it's okay (although I still argue the credential examples should be moved to STS).

Reply:
These were removed anyway.